pkg/engine/handlers/validation/validate_resource.go

```diff
@@ -183,15 +183,19 @@ func (v *validator) validateOldObject(ctx context.Context) (*engineapi.RuleRespo
 	return resp, nil
 }
 
+// This function gets called
 func (v *validator) validateForEach(ctx context.Context) *engineapi.RuleResponse {
 	applyCount := 0
 	for _, foreach := range v.forEach {
+		//From this we get the part of the resource that corresponds to the jmespath query present in the forEach block in the policy
```

Comment:
same comment as above....the comment seems unnecessary and redundant with the code.